controllers/articlesController.js

```diff
@@ -72,3 +73,15 @@ exports.updateArticlesVotes = (req, res, next) => {
       next(err);
     });
 };
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+
+  deleteCommentByIdInDB(comment_id)
+    .then((deleted) => {
+      res.status(204).send({ deleted });
```

Comment:
This should not send back a response body, please change this to - res.status(204).send() 👍